x/auction/types/auctions.go

```diff
@@ -111,117 +58,64 @@ func (a BaseAuction) String() string {
 	)
 }
 
-// NewBaseAuction creates a new base auction
-func NewBaseAuction(seller sdk.AccAddress, lot sdk.Coin, initialBid sdk.Coin, EndTime EndTime) BaseAuction {
-	auction := BaseAuction{
-		// no ID
-		Initiator:  seller,
-		Lot:        lot,
-		Bidder:     seller,     // send the proceeds from the first bid back to the seller
-		Bid:        initialBid, // set this to zero most of the time
-		EndTime:    EndTime,
-		MaxEndTime: EndTime,
-	}
-	return auction
-}
-
 // ForwardAuction type for forward auctions
```

Comment:
After the go doc comment (next line so it doesn't get included in the api docs), I would include a quick note of when each auction is used.